src/nl/rubensten/texifyidea/window/PreviewForm.kt

```diff
@@ -0,0 +1,126 @@
+package nl.rubensten.texifyidea.window
+
+import com.google.common.io.Files
+import java.io.*
+import java.util.concurrent.TimeUnit
+import javax.imageio.ImageIO
+import javax.swing.*
+
+class PreviewForm {
+    var panel: JPanel? = null
+    private var equation_area: JTextArea? = null
+    private var output_area: JTextArea? = null
+    private var output_panel: JPanel? = null
+    private var equation_panel9: JPanel? = null
+    private var preview_panel: ImagePanel? = null
+
+    private val preamble = """
+    \usepackage{amsmath,amsthm,amssymb,amsfonts}
+    \usepackage{color}
+    """.trimIndent()
+
+    fun set_equation_text(equation_text: String) {
```

Comment:
rename to camelCase, also rename parameter to camelCase

Maybe add KDoc to the method.

Also, split this baby up. It's way too long :P